packages/main/src/Carousel.hbs

```diff
@@ -15,7 +15,8 @@
 				<div id="{{id}}"
 					class="ui5-carousel-item {{classes}}"
 					style="{{styles}}"
-					role="option"
+					role="listitem"
+					aria-roledescription={{_roleDescription}}
```

Comment:
move the `aria-roledescription={{_roleDescription}}` to div where the role="list"